src/editor/EditorOptionHandlers.js

```diff
@@ -96,6 +98,7 @@ define(function (require, exports, module) {
     CommandManager.register(Strings.CMD_TOGGLE_LINE_NUMBERS, Commands.TOGGLE_LINE_NUMBERS, _getToggler(SHOW_LINE_NUMBERS));
     CommandManager.register(Strings.CMD_TOGGLE_ACTIVE_LINE, Commands.TOGGLE_ACTIVE_LINE, _getToggler(STYLE_ACTIVE_LINE));
     CommandManager.register(Strings.CMD_TOGGLE_WORD_WRAP, Commands.TOGGLE_WORD_WRAP, _getToggler(WORD_WRAP));
+    CommandManager.register("Always show full file path", Commands.TOGGLE_ALWAYS_SHOW_FULL_FILE_PATH, _getToggler(ALWAYS_SHOW_FULL_FILE_PATH));
```

Comment:
Should be `Strings.CMD_ALWAYS_SHOW_FULL_FILE_PATH`

Reply:
I was going to ask about that, as I'm not sure about all the translations, shall I just add a placeholder in the ones I'm not sure about?

Reply:
Translations fall back to english if they are not specified so it's okay to leave them alone 👍

Reply:
Okay brilliant, thanks. 

I'm just finishing off the unit testing so should be done very soon